src/backend/launcher.ts

```diff
@@ -393,24 +393,6 @@ function setupWineEnvVars(
   if (gameSettings.enableDXVKFpsLimit) {
     ret.DXVK_FRAME_RATE = gameSettings.DXVKFpsCap
   }
-  if (
-    gameSettings.showMangohud &&
-    !gameSettings.enviromentOptions.find(
-      ({ key }) => key === 'MANGOHUD_CONFIGFILE'
-    )
-  ) {
-    if (!process.env.XDG_CONFIG_HOME) {
-      ret.MANGOHUD_CONFIGFILE = join(
-        flatPakHome,
-        '.config/MangoHud/MangoHud.conf'
-      )
-    } else {
-      ret.MANGOHUD_CONFIGFILE = join(
-        process.env.XDG_CONFIG_HOME,
-        'MangoHud/MangoHud.conf'
-      )
-    }
```

Comment:
this was a workaround because we were setting XDG_CONFIG_HOME originally, and now we aren't so this is no longer needed